tst/testinstall/package.tst

```diff
@@ -418,8 +418,17 @@ false
 gap> IsPackageLoaded("mockpkg", ">=2.0");
 false
 
-#
-gap> mockpkgpath := DirectoriesLibrary("tst/mockpkg")[1];;
+# load mockpkg via a symlink in a directory called `pkg`
+# so we can test ExtendRootDirectories below
+# first create a temporary directory for all of this
+gap> tmp_dir := DirectoryTemporary( );;
+
+# create a subdirectory `<tmp_dir>/pkg`
+gap> Exec( Concatenation( "mkdir -p ", Filename( tmp_dir, "/pkg" ) ) );
+
+# make `<tmp_dir>/pkg/mockpkg` a symlink to `tst/mockpkg`
+gap> Exec( Concatenation( "ln -sfn ", Filename( DirectoriesLibrary("tst/mockpkg")[1], "" )," ", Filename( tmp_dir, "pkg/mockpkg" ) ) );
```

Comment:
```suggestion
gap> Exec( Concatenation( "ln -sfn ", Filename( DirectoriesLibrary("tst/mockpkg"), "" )," ", Filename( tmp_dir, "pkg/mockpkg" ) ) );
```
It should not be necessary to use `[1]` here.

Reply:
Done!